ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/about-you/about-you.component.html

```diff
@@ -3,10 +3,73 @@
     <h1 class="govuk-heading-xl" i18n="@@aboutYou_h1">
       About you (the legal representative)
     </h1>
-    <app-hearing-details-header></app-hearing-details-header>
-    <div class="mt-20">
-      <button class="govuk-button" (click)="continue()" i18n="@@aboutYou_btn_continue" id="continue"
-        type="button">Continue</button>
-    </div>
+
+    <form [formGroup]="form" (ngSubmit)="continue()">
+
+      <div id="form-container" [ngClass]="isFormInvalid && !isTextInputInvalid ? 'govuk-form-group--error' : 'govuk-form-group'">
+        <div class="govuk-fieldset">
+          <div *ngIf="isFormInvalid && !isTextInputInvalid">
+            <span class="govuk-error-message" i18n="@@aboutYou_span_1">
```

Comment:
the tag i18n , I believed , should be unique, as we have for individual the same tag, should we change it  by adding 'rep'

Reply:
Fixed